WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -67,8 +67,9 @@ def submit_report_nag(competition)
     )
   end
 
-  def results_submitted(competition, message, sender_name, file_contents)
+  def results_submitted(competition, schedule_url, message, sender_name, file_contents)
```

Comment:
Maybe it would be more handy to just pass `ResultSubmission` instance here?

Reply:
Good idea! Done.